features/graph/documentation/README.md

```diff
@@ -1,38 +1,82 @@
 
-Netlify Graph in its current state has three core components, that you can learn more about by reading related documentation pages:
+## Get started with Netlify Graph
 
-- [API Management](api-management.md)
-- [API Explorer](api-explorer.md)
-- [API Authentication](api-authentication.md)
+  1. [Link your local project directory to your Netlify site](authentication.md#link-your-local-project-directory-to-your-netlify-site)
+  2. [Connect to your first API or service](authentication.md#connect-to-your-first-api-or-service) in the Netlify UI.
+  3. [Start a local CLI session with Netlify Dev](authentication.md#start-a-local-cli-session-with-netlify-dev).
+  4. [Create a GraphQL query with Graph Explorer](authentication.md#create-a-graphql-query-with-graph-explorer).
+  5. [Generate a query handler](authentication.md#generate-a-query-handler).
+  6. [Use a query handler in your project](authentication.md#use-a-query-handler-in-your-project).
 
-## Enabling the feature
+## Queries, mutations, and subscriptions
+<!-- QUESTION: Are we going to address fragments? -->
+<!-- QUESTION: Is my assumption that these are standard GraphQL correct? -->
+You use standard GraphQL queries, mutations, and subscriptions to interact with your Netlify Graph data.
 
-To enable Netlify Graph in your account, you need to use Netlify Labs, where you can click on **Enable** next to **Netlify Graph**.
+- **[Queries](https://graphql.org/learn/queries/).** Are how you get data.
+- **[Mutations](https://graphql.org/learn/queries/#mutations).** Are how you modify data. 
+- **[Subscriptions](https://www.onegraph.com/docs/subscriptions.html).** Let you receive data in response to an event.
 
-![Enabling Netlify Graph in Netlify Labs](../../../media/graph/enable-labs.gif)
+<!-- TODO: Possibly add examples, though I think linking to external sources is probably the best way to go. If we want to expand this section, we could have subsections for each type. If we want to spend the time on it and see value in it, we could also have a separate page dedicated to more in-depth explanation of these. -->
 
-Once enabled, you will notice a new **Graph** tab in the top navigation bar when you go to any of your sites on Netlify.
+## Access secrets with Netlify Graph
 
-![Top nav bar showing Netlify Graph](../../../media/graph/graph-top-nav.png)
+You can access secrets for all of the APIs and services you connect to Netlify Graph. Check out the [Authentication docs](/authentication.md#basic-secret-handling) for details.
+
+## Netlify Graph and Next.js
+
+If you are building a Next.js application, you can automatically generate Netlify Graph bindings by running the Netlify Graph commands inside your application folder.
+
+<!-- QUESTION: perhaps this is my unfamiliarity with Next.js, but what's the difference between the application root and the repository root? They're not the same? -->
```

Comment:
@localden, just want to make sure that the following note is still accurate.

Reply:
Yes, that should still be accurate @KyleBlankRollins